utils/lists.py

```diff
@@ -6,7 +6,26 @@
     "You'll be the judge", 'no... (╯°□°）╯︵ ┻━┻', 'no... baka',
     'senpai, pls no ;-;'
 ]
+i = 0
+answers = []
+questions = []
 
+with open ("/Users/sethsarchet/Desktop/QuestionList.txt") as fp:
+    count = 0
+    a_count = 0
+    q_count = 0
+    
+    for line in fp:
+        count +=1
+        if count % 2 == 0: # answer
+            answers.append(line)
+            print(answers[a_count])
+            a_count+=1
+        else: # question
+            questions.append(line)
+            print(questions[q_count])
+            q_count+=1
```

Comment:
This section of code is more complex than it needs to be. A simple while loop that checks for EOF as it's terminating condition with two reads in the middle would have been much simpler.